### ISSUE TITLE
Version 4.0 failed to install on Windows

### ISSUE THREAD

**Issue by daniser** [spatie/laravel-data#653]
**✏️ Describe the bug**
Installation fails on Windows.
```
In ZipDownloader.php line 225:

  The archive may contain identical file names with different capitalization (which fails on case insensitive filesystems): ZipArchive::extractTo(D:\OSPanel\
  domains\ttb/vendor/composer/dfbb640e/spatie-laravel-data-fa428ce/lint-staged.config.js ): Failed to open stream: Permission denied


In ZipDownloader.php line 212:

  ZipArchive::extractTo(D:\OSPanel\domains\ttb/vendor/composer/dfbb640e/spatie-laravel-data-fa428ce/lint-staged.config.js ): Failed to open stream: Permissio
  n denied
```

**↪️ To Reproduce**
Require/update this package on Windows:
```sh
composer require spatie/laravel-data ^4.0
```

**🖥️ Versions**

Windows: 10.0.19045.3930
PHP: 8.1.26
Composer: 2.7.0
Laravel Data: 4.0.0

**Comment by rubenvanassche** [comment]
Seems like a permission issue on windows: https://stackoverflow.com/questions/72557472/error-symfony-installation-the-archive-may-contain-identical-file-names-with-d

**Comment by daniser** [comment]
@rubenvanassche Actually I found the problem:
https://github.com/spatie/laravel-data/blob/main/lint-staged.config.js%20
This file has space in its name after .js extension.

**Comment by rubenvanassche** [comment]
Weird, good catch! Fixed it, will be released with data v4.0.1 later today.
